Allow to disable deployments (#3570)
woodpecker-ci/woodpecker: docs/docs/20-usage/71-project-settings.md

Enables handling webhook's pull request event. If disabled, then pipeline won't run for pull requests.

## Allow deployments

> Allows you to start a pipeline with `deploy` event from a succeeded pipeline.

Reviewer: ```suggestion
Enables a pipeline to be started with the `deploy` event from a successful pipeline.
```

:::danger

> Only enable this option if you trust all users that have push access to your repository.
> Otherwise these users are able to steal secrets only available to `deploy` events.

Reviewer: ```suggestion
Only activate this option if you trust all users who have push access to your repository.
Otherwise, these users will be able to steal secrets that are only available for `deploy` events.
```
